sys/shell/cmds/cord_ep.c

```diff
@@ -26,9 +26,12 @@
 #include "net/gnrc/netif.h"
 #include "net/nanocoap.h"
 #include "net/sock/util.h"
+#include "uri_parser.h"
 #include "shell.h"
 
-static int make_sock_ep(sock_udp_ep_t *ep, const char *addr)
+#define COAPS_PORT 5684
```

Comment:
My code was preventing using `coaps` in the uri scheme. This is fixed now. However: 
I'm not even sure if the underlying code supports coaps? But in any case, a shell command should not lock to only supporting one over the other?
Additionally, there was no `#define COAPS_PORT 5684` anywhere in RIOT. Why? e.g. `coap.h` where `COAP_PORT` is defined is described as `Generic CoAP values as defined by RFC7252`. This RFC also defines the uri scheme `coaps://` and states it should hint the default port `5684`. 

This file is not the right place for such a define. What am I missing? Feels like an user error on my side.